code/ship/ship.cpp

```diff
@@ -5055,6 +5061,26 @@ void ship_parse_post_cleanup()
 		{
 			Warning(LOCATION, "$Rotation time: too low; this will disable rotational collisions. All three variables should be >= 1.39.\nFix this in ship '%s'\n", sip->name);
 		}
+
+		// ensure player min velocity makes sense
+		if (sip->min_vel.xyz.x > sip->max_vel.xyz.x || sip->min_vel.xyz.x < 0.0f)
+		{
+			Warning(LOCATION, "$Player Minimum Velocity X-value (%f) is negative or greater than max velocity X-value (%f), setting to zero\nFix for ship '%s'\n",
```

Comment:
It would be better to use `error_display` here because that will also output the table file name and line number.